lib/ecto/query/builder.ex

```diff
@@ -218,6 +219,12 @@ defmodule Ecto.Query.Builder do
     error! "`#{Macro.to_string(other)}` is not a valid query expression"
   end
 
+  def runtime_fragment!([{_,_}|_] = kw) when is_list(kw), do: kw
```

Comment:
Shall we consider empty list here as input?
We could use `Keyword.keyword?/1` function, don't think we expect really long lists there.